ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +250,73 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Operation Throughput:
+                        units: operations/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Reads:
+                                dpName: diskstats_readsCompleted
+                                lineType: LINE
+                            Writes:
+                                dpName: diskstats_writesCompleted
+                                lineType: LINE
+                    Merge Rate:
+                        units: operations merged/sec
+                        miny: 0
+                        maxy: 100
```

Comment:
`maxy` shouldn't be set on operations merged/sec. It could definitely go above 100.